twitter_scraper.py

```diff
@@ -16,15 +16,18 @@ def gen_tweets(pages):
 
         while pages > 0:
             d = pq(r.json()['items_html'])
-
-            tweets = [tweet.text for tweet in d('.tweet-text')]
+            tweets = [tweet.text_content() for tweet in d('.tweet-text')]
             last_tweet = d('.stream-item')[-1].attrib['data-item-id']
 
             for tweet in tweets:
+                if 'http' in tweet:
+                    tweet = tweet.split('http',1)
+                    tweet = tweet[0] + ' http' + tweet[1]
                 if tweet:
                     yield tweet
 
-            r = requests.get(url, params={'max_position': last_tweet}, headers=headers)
+            r = requests.get(
+            url, params={'max_position': last_tweet}, headers=headers)
```

Comment:
I think its better that this line stays indented

Reply:
Yeah, sorry didn't mean for that to happen!

Reply:
No worries :+1: